src/main/java/frc/robot/subsystems/scoring/AimerIORoboRio.java

```diff
@@ -235,6 +238,19 @@ public void applyOutputs(AimerOutputs outputs) {
 
         request = motionMagicVoltage;
 
+        if (goalAngleRot
```

Comment:
As written, this would slowly bring the aimer back to the intake location. We only want to apply this small output if the goal AND the aimer position are both near the hard stop

However this might mess with long shots as written. So maybe rather than using the goalAngleRot we could set a flag for if we want to drive into the hard stops and turn that on and off from shooter subsystem

Reply:
Should I change this to be activated for state == intake when goalAngleRotation is close to zero and actual angle is close to zero, then? Or would it be better to create a new flag of its own?

Reply:
I'd prefer a new flag just to decouple it from states.

For example we may want to set this flag true in idle as well